packages/origin-ui/src/__tests__/utils/demo.ts

```diff
@@ -0,0 +1,225 @@
+        web3,
+        userContracts.UserContractLookup,
+        adminPK
+    );
+    const originContracts: any = await migrateCertificateRegistryContracts(
+        web3,
+        assetContracts.AssetContractLookup,
+        adminPK
+    );
+    const marketContracts: any = await migrateMarketRegistryContracts(
+        web3,
+        assetContracts.AssetContractLookup,
+        originContracts.OriginContractLookup,
+        adminPK
+    );
+
+    const deployResult = {
+        userContractLookup: '',
+        assetContractLookup: '',
+        originContractLookup: '',
+        marketContractLookup: '',
+        userLogic: '',
+        assetConsumingRegistryLogic: '',
+        assetProducingRegistryLogic: '',
+        certificateLogic: '',
+        marketLogic: ''
+    };
+
+    deployResult.userContractLookup = userContracts.UserContractLookup;
+    deployResult.assetContractLookup = assetContracts.AssetContractLookup;
+    deployResult.originContractLookup = originContracts.OriginContractLookup;
+    deployResult.marketContractLookup = marketContracts.MarketContractLookup;
+    deployResult.userLogic = userContracts.UserLogic;
+    deployResult.assetConsumingRegistryLogic = assetContracts.AssetConsumingRegistryLogic;
+    deployResult.assetProducingRegistryLogic = assetContracts.AssetProducingRegistryLogic;
+    deployResult.certificateLogic = originContracts.CertificateLogic;
+    deployResult.marketLogic = marketContracts.MarketLogic;
+
+    await axios.put(
+        `${API_BASE_URL}/OriginContractLookupMarketLookupMapping/${deployResult.originContractLookup.toLowerCase()}`,
+        {
+            marketContractLookup: deployResult.marketContractLookup.toLowerCase()
+        }
+    );
+
+    const userLogic = new UserLogic(web3, deployResult.userLogic);
+    const assetProducingRegistryLogic = new AssetProducingRegistryLogic(
+        web3,
+        deployResult.assetProducingRegistryLogic
+    );
+    const assetConsumingRegistryLogic = new AssetConsumingRegistryLogic(
+        web3,
+        deployResult.assetConsumingRegistryLogic
+    );
+    const certificateLogic = new CertificateLogic(web3, deployResult.certificateLogic);
+    const marketLogic = new MarketLogic(web3, deployResult.marketLogic);
+
+    const conf: IStoreState['configuration'] = {
+        blockchainProperties: {
+            activeUser: {
+                address: ACCOUNTS.ADMIN.address,
+                privateKey: adminPK
+            },
+            producingAssetLogicInstance: assetProducingRegistryLogic,
+            consumingAssetLogicInstance: assetConsumingRegistryLogic,
+            certificateLogicInstance: certificateLogic,
+            userLogicInstance: userLogic,
+            marketLogicInstance: marketLogic,
+            web3
+        },
+        offChainDataSource: {
+            baseUrl: API_BASE_URL
+        },
+        logger
+    };
+
+    const adminPropsOnChain: User.IUserOnChainProperties = {
```

Comment:
I suggest taking a look at Substitute mocking lib used for e.g in `market-matcher` for faster test objects generation https://github.com/energywebfoundation/origin/blob/2948e17dbba72b6e74e737b6a8b811b2234fbb42/packages/market-matcher/src/test/unit/MatchableAgreement.test.ts#L29

You would avoid creating those full-fledged objects yet with empty properties.